test/go.test.ts

```diff
@@ -32,19 +32,29 @@ suite('Go Extension Tests', () => {
 
 	test('Test Hover Provider', (done) => {
 		let provider = new GoHoverProvider();
-		let testCases: [vscode.Position, string][] = [
+		let printlnDoc = `Println formats using the default formats for its operands and writes to
+standard output. Spaces are always added between operands and a newline
+is appended. It returns the number of bytes written and any write error
+encountered.
+`
```

Comment:
Not sure I really like how templates strings break indentation, but it does not destroy the file's column count that way